components/gfx/font_cache_thread.rs

```diff
@@ -275,24 +275,7 @@ impl FontCache {
                             }
                             let bytes = mem::replace(&mut *bytes.lock().unwrap(), vec![]);
                             trace!("@font-face {} data={:?}", family_name, bytes);
-                            let bytes = match fontsan::process(&bytes) {
```

Comment:
fontsan was all sorts of problematic. It pulls in a bunch of c libs that dont compile properly.